app/src/main/java/com/realityexpander/tasky/agenda_feature/domain/Photo.kt

```diff
@@ -2,8 +2,8 @@ package com.realityexpander.tasky.agenda_feature.domain
 
 import android.net.Uri
 import android.os.Parcelable
-import com.realityexpander.tasky.agenda_feature.util.PhotoId
-import com.realityexpander.tasky.agenda_feature.util.UrlStr
+import com.realityexpander.tasky.agenda_feature.common.util.PhotoId
+import com.realityexpander.tasky.agenda_feature.common.util.UrlStr
```

Comment:
One tip that will make all your future code reviewers happy is to put such major name refactorings into a separate PR (which doesn't necessarily need a review), since it just adds noise to this PR.
I know it's annoying to always think about what to put in which PR, but the more you work with others the more important it gets that the project is easy to follow for everyone :) 
(I was also blamed way too often for big PRs btw, but now that I regularly review code I understand why 😅)

Reply:
OK, i will do that in the future.